backend/LexBoxApi/Services/EmailService.cs

```diff
@@ -48,6 +48,16 @@ public async Task SendForgotPasswordEmail(string emailAddress)
         await SendEmailAsync(email);
     }
 
+    public async Task SendNewAdminEmail(IAsyncEnumerable<User> admins, string newAdminName, string newAdminEmail)
+    {
+        await foreach (var admin in admins)
+        {
+            var email = StartUserEmail(admin);
+            await RenderEmail(email, new NewAdminEmail(admin.Name, newAdminName, newAdminEmail), admin.LocalizationCode);
+            await SendEmailAsync(email);
```

Comment:
rather than sending one email per user, let's just send 1 email and bcc each user. You'll need to look at `StartUserEmail` and just inline that code here, but that's preferable to rendering and sending many emails (today that would be 32 emails). We can just address them as "Admin" since we won't be able to customize the email per person.

Reply:
Done in commit d859031.